site/assets/scripts/app.js

```diff
@@ -268,7 +301,7 @@ function _updateChart(sorted) {
   return false;
 }
 
-function loadGeoLayer(layerKey) {
+function loadGeoLayer(layerKey, DataCatalog) {
```

Comment:
change the name of data catalog in this method... you are overriding the DataCatalog variable in the parent scope